CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Added
 
+- Self-hosted etcd will mount a hostPath volume directory defined as
+  value of annotation `selfhosted-etcd.alpha.coreos.com/checkpointdir`.
```

Comment:
I agree with this.
I also think we can combine the `varLock` volume into it as well. This is orthogonal. Nonetheless if it is OK. I will put up an issue for it.